package.json

```diff
@@ -1,24 +1,27 @@
 {
   "license": "CC0-1.0",
   "engines": {
-    "node": ">=12",
+    "node": ">=18",
     "npm": "6.x.x"
   },
   "scripts": {
     "lint": "eslint . --ignore-pattern public --ignore-pattern vendor --ignore-pattern app/assets/builds",
-    "build": "webpack"
+    "build": "webpack --config webpack.config.js",
+    "build:css": "build-sass app/assets/stylesheets/*.css.scss --out-dir=app/assets/builds"
   },
   "dependencies": {
+    "@18f/identity-build-sass": "^3.0.0",
     "@babel/core": "^7.23.2",
     "@babel/preset-env": "^7.23.2",
     "@rails/ujs": "^6.1.6",
     "babel-loader": "^9.1.3",
     "babel-plugin-polyfill-corejs3": "^0.8.0",
     "core-js": "^3.23.4",
     "fast-glob": "^3.2.11",
-    "identity-style-guide": "^2.2.3",
-    "webpack": "^5.76.0",
-    "webpack-cli": "^4.10.0"
+    "identity-style-guide": "3.0.0",
```

Comment:
3.0.0 was a pretty big jump for USWDS (from 2.0.3 to 2.9.0) and had a lot of breaking changes. Have you made a pass over those? For example, I'd expect to see some revisions to the base layout for banner, etc (SVG images, banner markup).

Reply:
I walked through the release notes from 2.0.3 to 2.9 and I think I got most if not everything!